lib/triplex.ex

```diff
@@ -281,14 +281,10 @@ defmodule Triplex do
   Returns the path for the tenant migrations on your `repo`.
   """
   def migrations_path(repo \\ config().repo) do
-    path =
-      repo.config()
-      |> Keyword.get(:priv, "priv/#{repo |> Module.split |> List.last |> Macro.underscore}")
-      |> Path.join("tenant_migrations")
-
-    repo.config()
-    |> Keyword.get(:otp_app)
-    |> Application.app_dir(path)
+    config = repo.config()
+    priv = config[:priv] || "priv/#{repo |> Module.split |> List.last |> Macro.underscore}"
+    app = Keyword.fetch!(config, :otp_app)
+    Application.app_dir(app, Path.join(priv, "tenant_migrations"))
```

Comment:
Not sure of the utility of this refactor here. Does the repo config changed to a map? I mean, I get that one could be confuse with the change of main value on the pipelines, is that the reason you did it?

PS.: Just out of curiosity, I think it got more readable anyway, so let's leave it your way.